packages/edge-demo-site/app/routes/index.tsx

```diff
@@ -0,0 +1,30 @@
+export function headers({ loaderHeaders, parentHeaders }: { loaderHeaders: Headers; parentHeaders: Headers }) {
+  return {
+    'Cache-Control': 'public, max-age=31536000, s-maxage=31536000',
```

Comment:
```suggestion
    'Cache-Control': 'public, max-age=31536000, s-maxage=31536000',
    'x-nf-runtime': 'Deno' in globalThis ? 'Edge' : 'Node',
```
This would be a useful addition for debugging. Maybe in the adapter itself actually, but definitely not a blocker!

Reply:
I'd probably opt for in the adapter for the `x-nf-runtime` header as this is just a demo site.